src/python/impactx/dashboard/Analyze/plot_PhaseSpaceProjections/phaseSpace.py

```diff
@@ -40,8 +40,22 @@ def run_simulation():
     """
     sim = ImpactX()
 
+    # space charge selections
+    if state.space_charge:
+        sim.max_level = state.max_level
+        sim.n_cell = state.n_cell
+        sim.particle_shape = state.particle_shape
+        sim.poisson_solver = state.poisson_solver
+        sim.space_charge = state.space_charge
+        sim.dynamic_size = state.dynamic_size
+        sim.prob_relative = state.prob_relative
+
+    # csr
+    if state.csr:
+        sim.csr = state.csr
+        sim.csr_bins = state.csr_bins
+
```

Comment:
Currently, `particle_shape` is only displayed when `space_charge` is set to True. However, I see that `particle_shape` is also being used in `run_chicane_csr.py`. Should we consider having `particle_shape` displayed for CSR as well?